### PR TITLE
[FIX] *: various pylint warnings

### PR DESCRIPTION
**Pull request by Feyensv**
Enterprise PR: https://github.com/odoo/enterprise/pull/25232

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/models.py [hunk 1]
```diff
@@ -4856,13 +4856,14 @@ def blacklist_given_fields(model):
 
         return [default]
 
-    def copy_translations(old, new, excluded=()):
+    def copy_translations(self, new, excluded=()):
         """ Recursively copy the translations from original to new record
 
         :param old: the original record
```
Comment (Julien00859) [inline odoo/models.py:4862]: should amend docstring too

File: odoo/addons/test_inherit/models.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def _compute_surname(self):
             if rec.field_in_mother:
                 rec.surname = rec.field_in_mother
             else:
-                super(mother, rec)._compute_surname()
```
Comment (Julien00859) [inline odoo/addons/test_inherit/models.py:52]: can we use py3-super style instead ?

**Review by Julien00859 — APPROVED** [review]
Good that we enable more stuff, thank you for you effort to fix everything :)

File: addons/website_sale_coupon/tests/test_shop_sale_coupon.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```
Comment (Williambraecky) [inline addons/website_sale_coupon/tests/test_shop_sale_coupon.py:1]: This file should be removed completely 🙂 
Thanks for taking the time to clean up the code 😄

Comment (Feyensv) [reply addons/website_sale_coupon/tests/test_shop_sale_coupon.py:1]: Ooopsie, seems like I missed something during the rebase :man_facepalming:
sorry for the ping :3

**Comment by Feyensv** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**